core/src/avm1/activation.rs

```diff
@@ -650,8 +650,8 @@ impl<'a, 'gc> Activation<'a, 'gc> {
     }
 
     fn action_clone_sprite(&mut self) -> Result<FrameControl<'gc>, Error<'gc>> {
-        let depth = self.context.avm1.pop();
-        let target = self.context.avm1.pop();
+        let depth = self.context.avm1.pop().coerce_to_f64(self)?;
```

Comment:
While reading related code, I noticed some more inaccuracies around `ActionCloneSprite` and `duplicateMovieClip()`. Opened #11228 for that.